src/app/account/AccountManager.java

```diff
@@ -51,6 +51,8 @@ public void readAccount() {
             if (getRegisteredAccounts() == null) {
                 setRegisteredAccounts(new LinkedHashMap<>());
             }
+        }catch ( FileNotFoundException e){
```

Comment:
Never not swallow exceptions like this. Either display the stack trace or an appropriate message.

On bigger systems, logging become very important so when exceptions are thrown and they are swallowed like this we will never see anything on the stack trace to tell us what happened. Always log something.